extension/app/ts/components/simulationExplaining/customExplainers/SimpleSendVisualisations.tsx

```diff
@@ -0,0 +1,139 @@
+		before: bigint
+		after: bigint
+	} | undefined
+}
+
+type TransferAddressParams = BeforeAfterAddress & {
+	renameAddressCallBack: RenameAddressCallBack
+	tokenOrEtherDefinition: DistributedOmit<TokenOrEtherParams, 'amount'>
+}
+
+export function AddressBeforeAfter({ address, beforeAndAfter, renameAddressCallBack, tokenOrEtherDefinition }: TransferAddressParams) {
+	return <>
+		<BigAddress
+			addressBookEntry = { address }
+			renameAddressCallBack = { renameAddressCallBack }
+		/>
+		{ beforeAndAfter === undefined ? <></> :
+			<span style = 'grid-template-columns: auto auto; display: grid; justify-content: space-between; margin-top: 10px'>
+				<span style = 'grid-template-columns: auto; display: grid;'>
+					<p class = 'paragraph' style = 'color: var(--subtitle-text-color);'> Before:</p>
+					<TokenOrEth { ... { ...tokenOrEtherDefinition, amount: beforeAndAfter.before } }/>
+				</span>
+				<span style = 'grid-template-columns: auto; display: grid;'>
+					<p class = 'paragraph' style = 'color: var(--subtitle-text-color);'> After:</p>
+					<TokenOrEth { ... { ...tokenOrEtherDefinition, amount: beforeAndAfter.after } }/>
+				</span>
+			</span>
+		}
+	</>
+}
+
+type SimpleSendParams = {
+	transaction: SimulatedAndVisualizedTransaction
+	asset: TokenOrEtherParams
+	sender: BeforeAfterAddress
+	receiver: BeforeAfterAddress
+	renameAddressCallBack: RenameAddressCallBack
+}
+
+function SimpleSend({ transaction, asset, sender, receiver, renameAddressCallBack } : SimpleSendParams) {
+	return <div class = 'notification transaction-importance-box'>
+		<span style = 'grid-template-columns: auto auto; display: grid;'>
+			<p class = 'paragraph' style = 'font-size: 28px; font-weight: 500; justify-self: right;'> Send&nbsp;</p>
+			<TokenOrEth
+				{ ...asset }
+				useFullTokenName = { false }
+				style = { { 'font-size': '28px', 'font-weight': '500' } }
+			/>
+		</span>
+		<p class = 'paragraph'> From </p>
+		<div class = 'box' style = 'background-color: var(--alpha-005); box-shadow: unset; margin-bottom: 0px;'>
+			<AddressBeforeAfter
+				{ ...sender }
+				renameAddressCallBack = { renameAddressCallBack }
+				tokenOrEtherDefinition = { asset }
+			/>
+		</div>
+		<p class = 'paragraph'> To </p>
+		<div class = 'box' style = 'background-color: var(--alpha-005); box-shadow: unset; margin-bottom: 0px;'>
+			<AddressBeforeAfter
+				{ ...receiver }
+				renameAddressCallBack = { renameAddressCallBack }
+				tokenOrEtherDefinition = { asset }
+			/>
+		</div>
+		<span class = 'log-table' style = 'grid-template-columns: min-content min-content min-content; margin-top: 5px;'>
+			<GasFee tx = { transaction } chain = { transaction.chainId } />
+		</span>
+	</div>
+}
+
+function getBeforeAndAfterBalanceForAddress(ethBalances: readonly EthBalanceChangesWithMetadata[], address: bigint) {
+	const filtered = ethBalances.filter((x) => x.address.address === address)
+	if (filtered.length === 0) {
+		return {
+			before: 0n,
+			after: 0n,
+		}
+	}
```

Comment:
What is the situation where this will be length 0?

Also, I wouldn't mind seeing this slimmed down, 6 lines seems excessive for a guard clause:
```suggestion
	if (filtered.length === 0) return { before: 0n, after: 0n }
```

Reply:
It should never happen, I could make it throw an error too